src/txdb.h

```diff
@@ -125,6 +125,19 @@ class CBlockTreeDB : public CDBWrapper
     bool LoadBlockIndexGuts(const Consensus::Params& consensusParams, std::function<CBlockIndex*(const uint256&)> insertBlockIndex);
 };
 
+class BaseIndexDB : public CDBWrapper
```

Comment:
A comment explaining this base class would be nice.

Reply:
Given that it gets moved to BaseIndex for better context in "index: Move index DBs into index/ directory.", would you still find the comment helpful?

Reply:
Right, ignore then.